task1/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.ForegroundColor = ConsoleColor.Green;
+            Console.WriteLine("Never gonna give you up");
+
+            Console.ForegroundColor = ConsoleColor.Yellow;
+            Console.WriteLine(@"We're no strangers to love
+You know the rules and so do I
+A full commitment's what I'm thinking of
+You wouldn't get this from any other guy");
+            Console.ForegroundColor = ConsoleColor.Gray;
```

Comment:
Лучше восстановить цвета консоли пользователя по умолчанию, а не ваши цвета.
```suggestion
            Console.ResetColor();
```